tools/parse/src/test/scala/org/allenai/nlpstack/parse/poly/polyparser/GoldParseTrainingVectorSourceSpec.scala

```diff
@@ -45,6 +45,10 @@ class GoldParseTrainingVectorSourceSpec extends UnitSpec {
   )
   val parseSource = InMemoryPolytreeParseSource(List(parse1))
 
+  /** TODO: These unit tests are currently disabled because the transition systems temporarily
+    * do not permit features to be specified during construction. They will be re-enabled when
+    * these functionality returns. */
+  /*
   "Calling TrainingVectorGenerator's generate" should "create parse1's feature vectors" in {
```

Comment:
You can ignore tests by saying `"Foo" should "bar" ignore { ... }`. Then at least they get compiled, and thus statically checked. It keeps the bit rot in check a little.